indexing-service/src/test/java/org/apache/druid/indexing/overlord/RemoteTaskRunnerTest.java

```diff
@@ -816,7 +816,7 @@ public void testBlacklistZKWorkers() throws Exception
 
     makeRemoteTaskRunner(rtrConfig);
 
-    TestRealtimeTask task1 = new TestRealtimeTask(
+    TestIndexTask task1 = new TestIndexTask(
         "realtime1",
```

Comment:
super nit-picky, but these task ids are kind of funny since they are no longer realtime tasks

Reply:
Oh totally overlooked them! Thanks for pointing out @clintropolis.